owner/src/main/java/org/the_chance/honeymart/ui/features/signup/SignupInteractionListener.kt

```diff
@@ -2,7 +2,7 @@ package org.the_chance.honeymart.ui.features.signup
 
 interface SignupInteractionListener {
     fun onClickLogin()
-    fun onClickSignup()
+    fun onClickContinue()
```

Comment:
Where is go to continue?

Reply:
This is a button-click action, not an event to name it like the continue-to-market info screen

Reply:
Need rename

Reply:
rename for what? it is a button within the screen what should it be?